### DISCUSSION TITLE
Bookmarks Backup

### DISCUSSION THREAD

**Discussion by oldphones68** [bromite/bromite#1836] [Q&A]
Suddenly I noticed the latest bookmarks backup size is significantly smaller than the previous backup. (650 kb old vs. 132 kb new).
However, I didn't delete any bookmarks.
Is this behavior normal or anything wrong here?

**Comment by csagan5** [comment]
Do you have the two files? Maybe you can compare them and figure this out.

**Reply by oldphones68** [reply]
I have imported the exported file in another phone.
They were restored properly.
